scenes/Restaurant/Customer.gd

```diff
@@ -57,6 +57,63 @@ func feed() -> void:
 		$AnimationPlayer.play("Eat")
 		show_food_effects(0.066)
 
+"""
+Recolors the customer according to the specified color definition. This involves updating shaders and sprite
+properties.
+
+Parameter: 'color_def' describes the sprites to recolor and how to recolor them.
+"""
+func recolor(color_def: Dictionary) -> void:
+	var line_color := Color("6c4331")
```

Comment:
if color_def lacks a "line" property, these various 'black' shader properties will retain their previous values. Nothing to fix necessarily, but could pose strange problems in the future.